docs/calicoctl/endpoint.md

```diff
@@ -162,6 +162,11 @@ calicoctl endpoint <ENDPOINT_ID> profile (append|remove|set) [--host=<HOSTNAME>]
     <PROFILES>: One or more profiles to append, remove, or set on the endpoint.
 ```
 
+This command perform the desired action even if the `<ENDPOINT_ID>` is specified 
```

Comment:
I'd word it so you can use identical text in both cases, so i'd remove the details about what the command does:

"The <ENDPOINT ID> uniquely identifies the endpoint.  Although it is not necessary to include the optional <HOSTNAME>, <ORCHESTRATOR_ID> and <WORKLOAD_ID> identifiers, this command executes faster and with reduced load on the etcd datastore when all of the identifiers are specified together.